controllers/managedocs_controller.go

```diff
@@ -439,6 +448,9 @@ func (r *ManagedOCSReconciler) reconcilePhases() (reconcile.Result, error) {
 		if err := r.reconcileOCSCSV(); err != nil {
 			return ctrl.Result{}, err
 		}
+		if err := r.reconcileAlertRelabelConfigSecret(); err != nil {
```

Comment:
Could you include a comment explaining what this secret is and why we are reconciling it? From this PR I can infer that it is used to cause the namespace label to be included in the alert. If I was just looking at the code, it would take some digging to figure this out.  I'm making this suggestion to ease readability.

Reply:
I'll add the comment explaining the need for this secret.